lib/commands/project/project.js

```diff
@@ -425,3 +457,13 @@ function testGenerate(directory, options, cb) {
     });
   });
 }
+
+function installDependencies(directory, message, cb) {
+  spawn('npm', ['install'], directory, function(err) {
+    if (err) {
+      emit('"npm install" failed. Please run "npm install" in %s.', directory);
+      return cb(err);
+    }
+    cb(null, message);
```

Comment:
its the callback that is passed to the functions that are calling this function. the success gets printed when you create the project b.c it has one passed to it, thats what prints the "success! you may now start your proc..."